lib/IRGen/TypeLayout.cpp

```diff
@@ -194,8 +194,8 @@ class LayoutStringBuilder {
       case RefCountingKind::Resilient: {
         uint64_t op = (static_cast<uint64_t>(refCounting.kind) << 56) | skip;
         B.addInt64(op);
-        B.addCompactFunctionReference(refCounting.metaTypeRef);
-        refCountBytes += sizeof(uint64_t) + sizeof(uint32_t);
+        B.add(refCounting.metaTypeRef);
```

Comment:
If it‘s relative, we can‘t resolve it in instantiated layout strings. If there‘s a better solution, I‘d be happy to change it.

Reply:
We have several options. Eventually, we will want to look through to the type layout string.

We could update the re-location when we build an instantiated string -- that would require parsing every string on instantiation.
We could resolve to the metadata pointer on instantiation. We would only have to do that once since every nested inclusion would already be resolved.
We could implement looking through to the type layout string (we can't always do that so we would need the previous one too)

Making the constant byte code strings be in writable memory would be unfortuntate.